.Lib9c.DevExtensions.Tests/Action/FaucetCurrencyTest.cs

```diff
@@ -25,12 +26,18 @@ public FaucetCurrencyTest(ITestOutputHelper outputHelper)
                 .WriteTo.TestOutput(outputHelper)
                 .CreateLogger();
 
-            _initialState = new Lib9c.Tests.Action.State();
-
 #pragma warning disable CS0618
             _ncg = Currency.Legacy("NCG", 2, null);
             _crystal = Currency.Legacy("CRYSTAL", 18, null);
 #pragma warning restore CS0618
+
+            var balance = ImmutableDictionary
+                .CreateBuilder<(Address Address, Currency Currency), FungibleAssetValue>();
+            balance.Add((GoldCurrencyState.Address, _ncg), _ncg * int.MaxValue);
+            _initialState = new Lib9c.Tests.Action.State(
+                balance: balance.ToImmutable()
+            );
```

Comment:
```suggestion
            var balance = ImmutableDictionary<(Address Address, Currency Currency), FungibleAssetValue>.Empty
                .Add((GoldCurrencyState.Address, _ncg), _ncg * int.MaxValue);
            _initialState = new Lib9c.Tests.Action.State(balance);
```

This may also work 🤔

Reply:
Oh, this is pretty neat!